sdk/plugin/plugin.go

```diff
@@ -49,6 +69,9 @@ type Info struct {
 	PluginType  Type  `yaml:",omitempty"`
 	PluginMods  []Mod `yaml:",omitempty"`
 
+	AssetParser            AssetParser
```

Comment:
Where will this be defined, in the yaml for the plugin or at the Optimus side

Reply:
this will be defined in yaml for the plugin, I forgot to write the yaml tagstruct there